github/with_appengine.go

```diff
@@ -17,10 +17,8 @@ package github
 import (
 	"context"
 	"net/http"
-
-	"google.golang.org/appengine"
 )
 
 func withContext(ctx context.Context, req *http.Request) (context.Context, *http.Request) {
-	return appengine.WithContext(ctx, req), req
+	return ctx, req
```

Comment:
Given this change, I think you can simplify `withContext` signature to return only `*http.Request` then. Can you?